### DISCUSSION TITLE
contribution starting point

### DISCUSSION THREAD

**Discussion by zfoobar** [nasa/fprime#1946] [General]
Hello - I'm a general software eng with no background in astrophysics, but I'm interested in contributing to build up my knowledge of software engineering in this field. Curious: what's the best way to start? I don't operate or own a spacecraft, so what would be the best ways to put the software through its paces in a lab environment and contribute something useful?

**Comment by LeStarch** [comment]
F´ doesn't necessarily require a background in astrophysics nor flight software. F´is a generic embedded systems framework and as such it can be run in the lab as you suggest.

Our tutorials and documentation point to generic ARM processors (like the raspberrypi) to run on the ground. This would be a good place to start with F´, try running it on ARM or something similar. Build a small application that runs on the laptop and then cross-compile to run on ARM.
